static/styles/settings.css

```diff
@@ -1192,6 +1192,7 @@ input[type="checkbox"] {
         color: hsl(0, 0%, 47%);
         min-width: 80px;
         line-height: 1.2;
+        transform: translateY(-14%);
```

Comment:
Can you explain why this change is correct?  Generally alignment changes involving magic numbers are fragile.

Reply:
the whole widget is not align with the text, and we cant apply centre style, the only thing i can figure out as of now is to push the widget above and just manually try to centre it this way